Makefile

```diff
@@ -161,13 +161,30 @@ PROTO_PATHS = \
 	$(GOPATH)/src/github.com/googleapis/googleapis \
 	$(GOPATH)/src/github.com/gogo/googleapis
 
+# [Warning]
+# The below directories have the no original implementation.
+# You should not add any features.
+# - internal/copress/gob
+# - internal/compress/gzip
+# - internal/compress/lz4
+# - internal/compress/zst
```

Comment:
```suggestion
# The below packages have no original implementation.
# You should not add any features.
# - internal/copress/gob
# - internal/compress/gzip
# - internal/compress/lz4
# - internal/compress/zstd
```

Reply:
I wonder which is the correct using 'directories' or 'packages' 🤔

Reply:
it is source code path so i think it is safe to use the wording `package` 🤔
what do you think?

Reply:
OK